hack/components/bump_linux_bridge.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source hack/components/common_functions.sh
+
+LINUX_BRIDGE_URL=$(cat components.yaml | shyaml get-value components.linux-bridge.url)
+LINUX_BRIDGE_COMMIT=$(cat components.yaml | shyaml get-value components.linux-bridge.commit)
+LINUX_BRIDGE_REPO=$(echo ${LINUX_BRIDGE_URL} | sed 's#https://\(.*\)#\1#')
+LINUX_BRIDGE_PATH=${GOPATH}/src/${LINUX_BRIDGE_REPO}
+
+echo 'Fetch linux-bridge sources'
+fetch_component ${LINUX_BRIDGE_PATH} ${LINUX_BRIDGE_URL} ${LINUX_BRIDGE_COMMIT}
+
+echo 'Get linux-bridge tag'
+LINUX_BRIDGE_TAG=$(get_component_tag ${LINUX_BRIDGE_PATH})
+
+echo 'Build container image with linux-bridge binaries'
+LINUX_BRIDGE_IMAGE=quay.io/kubevirt/cni-default-plugins
```

Comment:
generally speaking, maybe this worth to be a variable with default value, so one can override it, and bump the results also to his personal repository (need to be parameterized as well then of course), in order to create a custom bundle?

Reply:
IIUC there is only 1 image path at a certain point of time, and if it does changes ( like recently changed on multus) then it  should be changed in a proper PR. So I don't see a reason to make it a parameter that is changable from the outside

I mean if it was a common practice to use our own private images then maybe there would be a case for it, but from what I understood so far it's uncommon and not recommended